gradle/dependencies.gradle

```diff
@@ -1,7 +1,7 @@
 ext {
-    // https://android.googlesource.com/platform/prebuilts/sdk/+/refs/tags/android-13.0.0_r3/current/androidx/Android.bp#3901
-    ext.room_version = "2.4.0-alpha05"
-    // https://android.googlesource.com/platform/external/protobuf/+/refs/tags/android-13.0.0_r3/java/pom.xml#7
+    // https://android.googlesource.com/platform/prebuilts/sdk/+/refs/tags/android-13.0.0_r32/current/androidx/Android.bp#4586
+    ext.room_version = "2.5.0-alpha01"
```

Comment:
Too bad they include an early alpha for something so critical.

Reply:
Does this matter, as it has been used in AOSP for quite a while?

Reply:
It's not used extensively in AOSP.

See https://cs.android.com/search?q=androidx.room_room-runtime&start=1

From what we know, Google mostly checks in prebuilts.

@grote Happy to just check in a prebuilt from this in a separate change (and anything else that we don't want to rely on AOSP versions for - if they version is too old, or alpha, or has any other issue).

Reply:
Removed for unblocking it, added a comment instead

Reply:
Thanks for removing. I thought about it and actually there's probably no point in removing it. We build this app with AOSP, so whatever is in AOSP gets used. Removing this here just affects our gradle builds which are dev only.

So, the testing would actually need to happen when AOSP bumps its deps. Ideally, we'd also have some integration tests for K/V backups with room. E.g. Some sample data in the repo and we try to read and compare the result with what we expect. Then just running these tests (on-device!) would already give us more peace of mine.

Reply:
Opened an issue to track things here: https://github.com/seedvault-app/seedvault/issues/548